doc/release-notes/0.14/0.14.md

```diff
@@ -0,0 +1,151 @@
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.14 release notes
+
+:construction: These release notes are *work in progress* for the next release of OpenJ9.
+
+These release notes support the [Eclipse OpenJ9 0.14 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.14/plan).
+
+
+## Binaries and supported environments
+
+OpenJ9 release 0.14 supports OpenJDK 8, 11, and 12. Binaries are available at the AdoptOpenJDK project:
+
+- [OpenJDK 8 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=openj9)
+- [OpenJDK 11 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=openj9)
+- [OpenJDK 12 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk12&jvmVariant=openj9)
+
+All builds are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+
+:pencil: On Linux&reg; and AIX&reg; platforms, the OpenSSL 1.0.2 or 1.1.X library is expected to be found on the system path. If you use a package manager to install OpenSSL, the system path will be updated automatically. On other platforms, the OpenSSL 1.1.X library is currently bundled with the binaries from AdoptOpenJDK.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.14. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.14/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/5023">#5023</a></td>
+<td valign="top">Extended platform support for software-based pause-less garbage collection</td>
+<td valign="top">OpenJDK8 and later (Linux on POWER BE)</td>
+<td valign="top">Reduced garbage collection pause times when using -Xgc:concurrentScavenge with the gencon GC policy for Linux on POWER BE. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/227">#227</a></td>
```

Comment:
#227 was reverted, https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/275 is the replacement.

Note there is a separate PR for jdk11, which is https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/135

Reply:
Fixed